docker/build_windows.dockerfile

```diff
@@ -13,9 +13,12 @@ RUN apt-get update -y && \
 RUN git lfs install
 
 RUN echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections
-RUN DEBIAN_FRONTEND=noninteractive apt-get install -y \
+RUN RUN apt-get update -y && \
```

Comment:
Hi @benjaoming, there is duplicate `RUN command`.

Reply:
Sorry about this, just fixed now!